### ISSUE TITLE
Filtering causes Error: $rootScope:infdig Infinite $digest Loop

### ISSUE THREAD

**Issue by ernestjumbe** [handsontable/ngHandsontable#114]
If you do something like:
```html
<input type="text" ng-model="search.name.first">
<hot-table on-after-selection="ctrl.onAfterSelection" stretch-h="ctrl.stretchH" row-headers="true" col-headers="true" datarows="ctrl.data | filter:search:strict" columns="ctrl.columns"></hot-table>
```
You get `Error: $rootScope:infdig Infinite $digest Loop` With smaller data sets the filter will work however, with larger data sets the Browser becomes unresponsive when you begin to type in the input box.

**Comment by budnix** [comment]
Confirmed, Thanks!

**Comment by budnix** [comment]
Bug fixed in 0.7.0-beta3
